### DISCUSSION TITLE
Strange mania combo counter placement with Center anchor

### DISCUSSION THREAD

**Discussion by curoviyxru** [ppy/osu#29499] [Q&A] [answered]
Hi. I'm trying to edit Argon Pro skin for mania. I move combo counter, save skin and it resets. What am I doing wrong? It seems to be a problem with combo counter only, other components behave normal.
upd: it seems this happens with combo counter only with Center anchor only.

https://github.com/user-attachments/assets/ee64754d-df5c-41db-9357-d91942beec34

[compressed-logs.zip](https://github.com/user-attachments/files/16653284/compressed-logs.zip)

**Comment by peppy** [comment]
@frenzibyte

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/pull/29529
